pinot-broker/src/main/java/org/apache/pinot/broker/requesthandler/BaseBrokerRequestHandler.java

```diff
@@ -670,14 +670,16 @@ private void logBrokerResponse(long requestId, String query, RequestContext requ
     if (_queryLogRateLimiter.tryAcquire() || forceLog(brokerResponse, totalTimeMs)) {
       // Table name might have been changed (with suffix _OFFLINE/_REALTIME appended)
       LOGGER.info("requestId={},table={},timeMs={},docs={}/{},entries={}/{},"
-              + "segments(queried/processed/matched/consuming/unavailable):{}/{}/{}/{}/{},consumingFreshnessTimeMs={},"
+              + "segments(queried/processed/matched/consumingQueried/consumingMatched/consumingProcessed/unavailable):"
```

Comment:
The ordering is incorrect
```suggestion
              + "segments(queried/processed/matched/consumingQueried/consumingProcessed/consumingMatched/unavailable):"
```